CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 # Changelog
 
+## 0.8.0
+
+Check out the [blog post](https://astral.sh/blog/ruff-v0.8.0) for a migration guide and overview of the changes!
+
+### Breaking changes
+
+See also, the "Remapped rules" section which may result in disabled rules.
+
+- **Default to Python 3.9**
+
+    Ruff now defaults to Python 3.9 instead of 3.8 if no explicit Python version is configured using `ruff.target-version` or `project.requires-python` ([#13896](https://github.com/astral-sh/ruff/pull/13896))
+
+- **Use XDG (i.e. `~/.local/bin`) instead of the Cargo home directory in the installer**
+
+    Previously, Ruff's installer used `$CARGO_HOME` or `~/.cargo/bin` for its target install directory. Now, Ruff will be installed into `$XDG_BIN_HOME`, `$XDG_DATA_HOME/../bin`, or `~/.local/bin` (in that order). Note, this only applies if you installed Ruff with the standalone installer (using the shell or PowerShell script but not when using e.g. pip).
+
+- **Updated Unicode standard**
+
+    Ruff now uses a more recent version of the Unicode standard. In very rare cases, this can lead to formatting changes or new `E501` violations if the computed line width differs.
```

Comment:
```suggestion
    Ruff now uses a more recent version of the Unicode standard. In very rare cases, this can lead to formatting changes or new [`E501`](https://docs.astral.sh/ruff/rules/line-too-long/) violations if the computed line width differs.
```